src/Events/Global/WalletCreated.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Enjin\Platform\Events\Global;
+
+use Enjin\Platform\Events\PlatformBroadcastEvent;
+use Enjin\Platform\Traits\HasCustomQueue;
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Database\Eloquent\Model;
+
+class WalletCreated extends PlatformBroadcastEvent
+{
+    use HasCustomQueue;
+
+    /**
+     * Create a new event instance.
+     */
+    public function __construct(Model $wallet)
+    {
+        parent::__construct();
+
+        $this->model = $wallet;
+
+        $this->broadcastData = [
+            'id' => $wallet->id,
+            'publicKey' => $wallet->public_key,
+        ];
+        $this->broadcastChannels = $wallet->public_key
+            ? [new Channel($wallet->public_key)]
+            : [];
```

Comment:
Do we also need to broadcast on a more general channel, like the Platform App channel, otherwise how will this event be picked up?  Will people always know the public key of a wallet before it's created?